### ISSUE TITLE
Check for non-indentation tabs in haml

### ISSUE THREAD

**Issue by agrimm** [brigade/haml-lint#153]
The following, which contains a tab, doesn't raise an error in haml-lint:

```
%td Hello
```

Although it probably doesn't cause any errors in HAML, I want to avoid using tabs unless there's a good reason I'm using them.

Should haml-lint have a linter for this?
